src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ReleaseService.cs

```diff
@@ -337,7 +338,12 @@ public class ReleaseService : IReleaseService
                         .OnSuccess(async () =>
                         {
                             _context.Releases.Update(release);
-                            await _context.AddAsync(releaseStatus);
+
+                            if (oldStatus != request.ApprovalStatus)
```

Comment:
I think an approver can change the release note when a Release is already scheduled.  I think this change might stop this or force them to unapprove first before reapproving with a different note so worth checking it.

I.e, they won't be able to do this, and the change would be silently ignored I think? 

![image](https://user-images.githubusercontent.com/4147126/123134390-a1aaef00-d448-11eb-948a-fe0b93f8c562.png)

Reply:
We discussed this issue on standup and it was decided that a new endpoint would be created to update the release from the sign off page. See EES-2448 ticket on Jira.